README.md

```diff
@@ -0,0 +1,40 @@
+# lsst-rsp
+
+User-facing Python classes and functions for use in the RSP environment.
+Learn more at https://lsst-rsp.lsst.io
+
+Install from PyPI:
+
+```sh
+pip install lsst-rsp
```

Comment:
--user right?

Reply:
Yeah.  More boilerplate.  Probably I should combine this with the next suggestion and create a section for "how to install this in the RSP".

Reply:
Done.